daterangepicker.js

```diff
@@ -290,6 +291,11 @@
                 this.timePicker12Hour = options.timePicker12Hour;
             }
 
+            if (typeof options.autoApply === 'boolean') {
+                this.autoApply = options.autoApply;
+                this.container.find('.applyBtn, .cancelBtn').addClass('hide');
```

Comment:
Agreed. I didn't consider the case when autoApply is present but set to false! I'll fix it now.